prow/plugins/override/override.go

```diff
@@ -336,6 +351,29 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent,
 		return oc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, user, resp))
 	}
 
+	checkruns, err := oc.ListCheckRuns(org, repo, sha)
+	if err != nil {
+		resp := fmt.Sprintf("Cannot get commit checkruns for PR #%d in %s/%s", number, org, repo)
+		log.WithError(err).Warn(resp)
+		return oc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, user, resp))
+	}
+
```

Comment:
We have to de-duplicate them first or not?

Reply:
yes, done that